conversions/include/conversions/py_occupancy_grid.py

```diff
@@ -74,6 +74,18 @@ def get_extent(self):
         top_right = self.resolution*(np.flipud(self.shape)) + self.origin
         return self.origin[0], top_right[0], self.origin[1], top_right[1]
 
-    def plot(self):
-        """ Plots the grid as an image and returns a handle to the image axes """
-        return plt.imshow(self.grid, extent=self.get_extent())
\ No newline at end of file
+    def plot(self, ax=None, **kwargs):
+        """ Plots the grid as an image and returns a handle to the image axes
+        Args:
+            ax (matplotlib.axes): An optional axes handle. If present it will be modified
+                                  and returned. Otherwise a new axes object will be created.
+        """
+
+        im2 = np.zeros(self.grid.shape + (4,))
+        im2[:, :, 3] = np.flipud(self.grid)
+        if ax is None:
+            axesImage = plt.imshow(im2, extent=np.array(self.get_extent())-self.resolution/2)
+            return axesImage.axes
+        else:
+            ax.imshow(im2, extent=np.array(self.get_extent())-self.resolution/2)
+            return ax
```

Comment:
A few notes:
We need to nail down conventions including:

1. Where is the grid origin?
2. Where is the origin of each cell? (lower left? center?)
3. How are rows and columns of the grid related to the X,Y axes?
This will ensure our index_to_coord and coord_to_index methods are consistent with our plotting.

Might want to look here if we want to be consistent with costmap_2d:
https://github.com/ros-planning/navigation/blob/noetic-devel/costmap_2d/src/costmap_2d.cpp#L202